x-pack/plugins/beats_management/public/components/table/table_type_configs.tsx

```diff
@@ -107,7 +107,9 @@ export const BeatsTableType: TableType = {
         field: 'full_tags',
         name: 'Tags',
         options: uniq(
-          flatten(data.map((item: any) => item.full_tags)).map((tag: any) => ({ value: tag.id })),
+          flatten(data.map((item: any) => item.full_tags || [])).map((tag: any) => ({
```

Comment:
Isn't `full_tags` a required prop on `CMPopulatedBeats`? Shouldn't it default to an `[]` wherever it's being instantiated? Or am I showing my lack of TypeScript expertise by asking this?

Reply:
yes, but we are not loading those tags yet, this needs to be done yet in a future PR